README.md

```diff
@@ -1,2 +1,113 @@
-# frogtek.com-site
-Frogtek company main site
+# frogtek.com
```

Comment:
Try not to mix autogenerated code with your own changes in the same commit. This README went almost unnoticed by at least me...

Reply:
I mean, think in the reviewer when splitting up in commits. If 95% of the commit has no need to be reviewed again, since it is copied from another non-legacy and already reviewed commit, then extract that 5% brand new content so it doesn't get unnoticed by us (just for the future, anyway).